src/redux/reducers/ConceptReducers.js

```diff
@@ -24,15 +25,19 @@ import {
   EDIT_CONCEPT_CREATE_NEW_NAMES,
   EDIT_CONCEPT_REMOVE_ONE_NAME,
   REMOVE_CONCEPT,
-  QUERY_POSSIBLE_ANSWER_CONCEPTS,
   ADD_SELECTED_ANSWERS,
-  CHANGE_ANSWER_MAPPING,
+  REMOVE_SELECTED_ANSWER,
+  PRE_POPULATE_ANSWERS,
+  UNPOPULATE_PRE_POPULATED_ANSWERS,
+  ADD_NEW_ANSWER_ROW,
 } from '../actions/types';
 import {
   filterSources, filterClass, filterList, normalizeList, filterNames,
   filterDescriptions,
 } from './util';
 
+// const frontEndUniqueKey = uuid();
```

Comment:
What is the above used for?

Reply:
@dkayiwa I have taken out the commented out code.